locales/locales.php

```diff
@@ -1419,6 +1419,20 @@ public function __construct() {
 		$ml->google_code = 'ml';
 		$ml->facebook_locale = 'ml_IN';
 
+		$mlt = new GP_Locale(); 
+		$mlt->english_name = 'Maltese';
+		$mlt->native_name = 'Malti';
+		$mlt->lang_code_iso_639_1 = 'mt';
+		$mlt->lang_code_iso_639_2 = 'mlt';
+		$mlt->lang_code_iso_639_3 = 'mlt';
+		$mlt->country_code = 'mt';
+		$mlt->facebook_locale = 'mt_MT';
+		$mlt->google_code = 'mt';
```

Comment:
To make this consistent with other locales, please move the Google and Facebook codes to the end of the locale definition.